### PR TITLE
[15.0] [MIG] sale_order_report_without_price: Migrated the module in v15.

### PR DESCRIPTION
**Pull request by Chandresh-OSI**

### CODE REVIEW

**Comment by Chandresh-OSI** [comment]
Ping @patrickrwilson

**Review by patrickrwilson — APPROVED** [review]
Tested, works great. LGTM

File: sale_order_report_without_price/report/sale_report_templates.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        primary="True"
+        priority="99"
+    >
+        <xpath expr="//th[@name='th_priceunit']" position="replace" />
+        <xpath expr="//th[@name='th_discount']" position="replace" />
+        <xpath expr="//th[@name='th_taxes']" position="replace" />
+        <xpath expr="//th[@name='th_subtotal']" position="replace" />
+
+        <xpath expr="//td[@name='td_priceunit']" position="replace" />
+        <xpath expr="//td[count(@name) = 0]" position="replace" />
```
Comment (omar7r) [inline sale_order_report_without_price/report/sale_report_templates.xml:15]: You should get discount 'td' with [@t-if='display_discount'] for being more specific

**Review by hildickethan — APPROVED** [review]

**Pull request merged**